portal/src/main/webapp/config_service.jsp

```diff
@@ -140,7 +140,8 @@
              
         obj.put("oncoprintOncoKbHotspotsDefault",enableOncoKBandHotspots);    
         
-        obj.put("sessionServiceEnabled",GlobalProperties.getSessionServiceUrl()!= "");        
+        // we don't want to expose session service url to frontend so we need this flag
```

Comment:
the session service is always access by frontend via proxy.  that's why we don't need url.